js/dropdown.js

```diff
@@ -527,8 +529,25 @@
      * Place dropdown
      */
     _placeDropdown() {
+      /**
+       * Get closest ancestor that satisfies the condition
+       * @param {Element} el  Element to find ancestors on
+       * @param {Function} condition  Function that given an ancestor element returns true or false
+       * @returns {Element} Return closest ancestor or null if none satisfies the condition
+       */
+      const getClosestAncestor = function(el, condition) {
+        let ancestor = el.parentNode;
+        while (ancestor !== null && !$(ancestor).is(document)) {
+          if (condition(ancestor)) {
+            return ancestor;
+          }
+          ancestor = ancestor.parentNode;
+        }
+        return null;
+      };
+
```

Comment:
is this PR only moves `getClosestAnchor` from global to dropdown? And deletes `elementOrParentIsFixed`?  I don't see any other differences here... And I'm not sure how this fixes the linked issue...?

Reply:
Yes exactly!
**getClosestAnchor**
Because the function uses the $ operator, which is not accessible in the global.js File => This caused the Error.
So I searched where the function is used and it is only used by dropdown.js so I moved it there. This solves the Bug.
**elementOrParentIsFixed**
I saw that this function also uses the $ operatorm so I also search where the function is called. And it is never beeing called.
So I removed the function.
You can test it via the Select-Dropdown. Best regards and thanks for your time bro!

Reply:
So `elementOrParentIsFixed` was never documented and was some internal leftover?

Reply:
yes @DanielRuf I dont know why this function is there. I searched the whole repo and it wasnt used anywhere, so I removed it.